Dockerfile

```diff
@@ -20,24 +20,22 @@ RUN export DEBIAN_FRONTEND=noninteractive \
       libxslt-dev \
       pkg-config \
       python3-dev \
-      python3-pip \
       python3-venv \
-    && python3 -m venv /opt/netbox/venv \
-    && /opt/netbox/venv/bin/python3 -m pip install --upgrade \
-      pip \
-      setuptools \
-      wheel
+    && python3 -m venv --without-pip /opt/netbox/venv
```

Comment:
I believe UV manages the venv on its own.

Reply:
yep it does

Reply:
Yes, it could manage the venv, but I thought it would be a good idea to have the venv placed where we want it. Might be better if there are questions or something to debug.

Reply:
First replace move line 26 to line 2 (the uv COPY)

Then you can replace lines 23 and 24 with     
```Dockerfile
    && /usr/local/bin/uv venv /opt/netbox/venv --system

```